apps/editors/helpers.py

```diff
@@ -450,12 +450,19 @@ def get_actions(self, request, addon):
         if not addon.admin_review or acl.action_allowed(
                 request, 'ReviewerAdminTools', 'View'):
             if self.review_type != 'preliminary':
+                label = _lazy('Push to public')
+                if not addon.is_listed:
+                    label = _lazy('Grant full review')
```

Comment:
Just a nit but this is a case where I think a `if/else` would read better. No need to change unless you agree, tho.